### PR TITLE
Openapi endpoint implementation

### PR DESCRIPTION
**Pull request by chiararelandini**
Openapi endpoint at `/documentation/json`. By default it returns a yaml schema. To request JSON, the client must add the `Accept` header: as either `application/vnd.oai.openapi+json` or `application/json`.

### CODE REVIEW

File: src/main/java/eu/miaplatform/helidonHelloWorld/Main.java [hunk 1]
```diff
@@ -114,11 +116,17 @@ private static Routing createRouting(Config config) {
...
+                        .config(config)
+                        .webContext("/documentation/json")
+                        .build();
+
                 return Routing.builder()
                         .register(health)                   // Health at "/-/healthz"
                         .register(metrics)                  // Metrics at "/-/metrics"
                         .register(readiness)                // Readiness at "/-/ready"
                         .register("/hello", helloWorldService)
+                        .register(openapi)                  // Readiness at "/documentation/json"
```
Comment (frengooo) [inline src/main/java/eu/miaplatform/helidonHelloWorld/Main.java]:
```suggestion
                        .register(openapi)                  // Openapi documentation at "/documentation/json"
```

**Review by frengooo — APPROVED** [review]
lgtm

File: src/main/resources/META-INF/openapi.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+paths:
```
Comment (fredmaggiowski) [inline src/main/resources/META-INF/openapi.yml:1]: So API documentation must be statically written along side with code, there's no way to annotate the code (for example as in .netcore or springboot) or programmatically provide the schema (for example as in fastify)?

**Review by fredmaggiowski — APPROVED** [review]

File: src/main/resources/META-INF/openapi.yml (hunk 2 above)
Comment (chiararelandini) [reply src/main/resources/META-INF/openapi.yml:1]: From what I've seen, annotations can only be used in `Helidon MP`, which is a slightly different framework. In this [page](https://helidon.io/docs/latest/index.html#/se/openapi/01_openapi) all options for `Helidon SE` are described, but I think the best way is to include the static file `META-INF/openapi.yml` as a configuration.

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**